src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -299,6 +299,7 @@ class GardenVisitorFeatures {
             readToolTip(visitor, event.itemStack)
         }
 
+        toolTip.add("§aAccept Offer")
```

Comment:
Do we need this? the last lore should include this already

Reply:
Accept offer isn't the lore. it's the title, and it got removed before this pr